tests/std/tests/P0067R5_charconv/float_from_chars_test_cases.hpp

```diff
@@ -254,4 +254,529 @@ inline constexpr FloatFromCharsTestCase float_from_chars_test_cases[] = {
+        0x1.a87124p+107f},
+    {/* 0x1.38ff8fp+108 in decimal: */ "396772452133553939418920234516480", chars_format::general, 33, errc{},
+        0x1.38ff90p+108f},
+    {/* 0x1.13c1b5p+109 in decimal: */ "699126211644530005899810908930048", chars_format::general, 33, errc{},
+        0x1.13c1b4p+109f},
+    {/* 0x1.55eee7p+110 in decimal: */ "1733807367140220814406037893283840", chars_format::general, 34, errc{},
+        0x1.55eee8p+110f},
+    {/* 0x1.68ea01p+111 in decimal: */ "3660103578413974514481484236062720", chars_format::general, 34, errc{},
+        0x1.68ea00p+111f},
+    {/* 0x1.a20bfdp+112 in decimal: */ "8478997023821539945225867390615552", chars_format::general, 34, errc{},
+        0x1.a20bfcp+112f},
+    {/* 0x1.58562fp+113 in decimal: */ "13967954142855725924178258573656064", chars_format::general, 35, errc{},
+        0x1.585630p+113f},
+    {/* 0x1.8e458dp+114 in decimal: */ "32311637611412935510718602954145792", chars_format::general, 35, errc{},
+        0x1.8e458cp+114f},
+    {/* 0x1.635bb5p+115 in decimal: */ "57660169510975349749437807131099136", chars_format::general, 35, errc{},
+        0x1.635bb4p+115f},
+    {/* 0x1.019409p+116 in decimal: */ "83588925144890861008382505730965504", chars_format::general, 35, errc{},
+        0x1.019408p+116f},
+    {/* 0x1.b0eb7bp+117 in decimal: */ "280981044275986616573540753022648320", chars_format::general, 36, errc{},
+        0x1.b0eb7cp+117f},
+    {/* 0x1.a072a1p+118 in decimal: */ "540580110894867345113842255044542464", chars_format::general, 36, errc{},
+        0x1.a072a0p+118f},
+    {/* 0x1.211b6fp+119 in decimal: */ "750565105750951431544116449153908736", chars_format::general, 36, errc{},
+        0x1.211b70p+119f},
+    {/* 0x1.e54565p+120 in decimal: */ "2519671464696457305794640027949465600", chars_format::general, 37, errc{},
+        0x1.e54564p+120f},
+    {/* 0x1.93c0a7p+121 in decimal: */ "4192806175473153074327132029524639744", chars_format::general, 37, errc{},
+        0x1.93c0a8p+121f},
+    {/* 0x1.f33db1p+122 in decimal: */ "10368829531117867053281330031553937408", chars_format::general, 38, errc{},
+        0x1.f33db0p+122f},
+    {/* 0x1.f97c0fp+123 in decimal: */ "20997008966187027788044272068598956032", chars_format::general, 38, errc{},
+        0x1.f97c10p+123f},
+    {/* 0x1.46fa2fp+124 in decimal: */ "27164209632110478318892728203406737408", chars_format::general, 38, errc{},
+        0x1.46fa30p+124f},
+    {/* 0x1.73d8d9p+125 in decimal: */ "61783690480066413003444625390581055488", chars_format::general, 38, errc{},
+        0x1.73d8d8p+125f},
+    {/* 0x1.a19fe7p+126 in decimal: */ "138779583669858850032422956217074188288", chars_format::general, 39, errc{},
+        0x1.a19fe8p+126f},
+    {/* 0x1.f0b0b5p+127 in decimal: */ "330107300636279331787628201852738207744", chars_format::general, 39, errc{},
+        0x1.f0b0b4p+127f},
+

```

Comment:
Conventionally, we don't put empty lines immediately inside delimiters. (Not worth resetting testing, but I had to find _something_ to complain about.)